flathub.json

```diff
@@ -0,0 +1,7 @@
+{
+  "only-arches": [
+    "x86_64"
+  ],
+  "automerge-flathubbot-prs": true,
+  "disable-external-data-checker": true
```

Comment:
If you want to disable the global external-data-checker run, you need to provide your own GitHub Actions workflow to run it.

Reply:
Thank you for spotting this, it got carried over from Brave flatpak without me properly understanding what it was about. Removed to enable the global external-data-checker.